src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
```

Comment:
I was just wondering, would it be slightly better to leave a line to separate the imports from diff packages iirc 2030S used to do this... Not sure if its needed here.

```suggestion
import java.io.IOException;

import java.util.Scanner;

```

Reply:
Edit to above comment:

After running the checkstyle feature, I think the above suggestion is invalid for most **import** situations! Do take note when there's a need for it.